consult.el

```diff
@@ -1950,6 +1954,7 @@ See `multi-occur' for the meaning of the arguments BUFS, REGEXP and NLINES."
                                           'fontify)
                (point-marker) line)
               candidates)
+        (put-text-property 0 1 'consult-level (funcall level-function) (car candidates))
```

Comment:
Just for the record, this is a tricky line, because it depends on the match data from the previous `re-search-forward`.